testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
```

Comment:
DynaConf recommends to use DynaConf as a base class, as:

```suggestion
from dynaconf import Dynaconf
```